src/cache/gcs.rs

```diff
@@ -572,7 +572,7 @@ impl Storage for GCSCache {
         let data = entry.finish()?;
 
         let bucket = self.bucket.clone();
-        let response = bucket
+        let _response = bucket
```

Comment:
Are you sure this is ok? We are not evaluating the response. This might contain information on the caching attempt, so just ignoring it is wrong.

Reply:
Made use explicit in https://github.com/paritytech/sccache/pull/29/commits/ee16df7ba9cd3dbbe162b692933b98f234fa8476